crates/rust-analyzer/src/config.rs

```diff
@@ -46,84 +52,67 @@ use crate::{
 config_data! {
     struct ConfigData {
         /// Placeholder for missing expressions in assists.
-        assist_exprFillDefault: ExprFillDefaultDef              = "\"todo\"",
-        /// How imports should be grouped into use statements.
-        assist_importGranularity |
-        assist_importMergeBehavior |
-        assist_importMergeBehaviour: ImportGranularityDef  = "\"crate\"",
-        /// Whether to enforce the import granularity setting for all files. If set to false rust-analyzer will try to keep import styles consistent per file.
-        assist_importEnforceGranularity: bool              = "false",
-        /// The path structure for newly inserted paths to use.
-        assist_importPrefix: ImportPrefixDef               = "\"plain\"",
-        /// Group inserted imports by the https://rust-analyzer.github.io/manual.html#auto-import[following order]. Groups are separated by newlines.
-        assist_importGroup: bool                           = "true",
-        /// Whether to allow import insertion to merge new imports into single path glob imports like `use std::fmt::*;`.
-        assist_allowMergingIntoGlobImports: bool           = "true",
-
-        /// Warm up caches on project load.
-        cache_warmup: bool = "true",
-
-        /// Show function name and docs in parameter hints.
-        callInfo_full: bool                                = "true",
+        assist_expressionFillDefault: ExprFillDefaultDef              = "\"todo\"",
 
         /// Automatically refresh project info via `cargo metadata` on
         /// `Cargo.toml` changes.
         cargo_autoreload: bool           = "true",
-        /// Activate all available features (`--all-features`).
-        cargo_allFeatures: bool          = "false",
```

Comment:
`allFeatures` has been merged into `features` for both `cargo` and `checkOnSave`, so now one can either pass `"all"` or `["feat", ...]` which will enable either `--all-features` or `--features feat,...` respectively